plugins.txt

```diff
@@ -137,7 +137,7 @@ ssh-agent:333.v878b_53c89511
 ssh-credentials:308.ve4497b_ccd8f4
 sshd:3.312.v1c601b_c83b_0e
 structs:325.vcb_307d2a_2782
-support-core:1356.vd0f980edfa_46
+support-core:1362.vf5fe14238165
```

Comment:
Internal changes for developers: https://plugins.jenkins.io/support-core/releases/#version_1362.vf5fe14238165